user/languages/nix.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Building a Nix Project
+layout: en
+permalink: /user/languages/nix/
+---
+
+### What This Guide Covers
+
+This guide covers build environment and configuration topics specific to Nix projects. Please make sure to read our [Getting Started](/user/getting-started/) and [general build configuration](/user/customizing-the-build/) guides first.
+
+<div id="toc"></div>
+
+### Community-Supported Warning
+
+Travis CI support for Nix is contributed by the community and may be removed
+or altered at any time. If you run into any problems, please report them in the
+[Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues/new?labels=community:nix)
+and cc @domenkozar @garbas and @matthewbauer .
+
+## Overview
+
+To set up Travis to provide the Nix environment, add the following line to `.travis.yml`:
+
+```yaml
+language: nix
+```
+
+This will install the Nix store and set up a basic single-user profile. The default channel for "nixpkgs" will be "nixpkgs-unstable". More information on configuring channels is provided in the manual below.
```

Comment:
I'd skip the first line "To set up..." and instead use

To install the Nix store and set up a basic single-user profile:

```
+```yaml
 +language: nix
 +```
The default channel for "nixpkgs" will be "nixpkgs-unstable". More information on configuring channels is provided in the manual below.